.github/workflows/fuzzing.yaml

```diff
@@ -0,0 +1,22 @@
+name: Fuzzing v3rpc
+on: [push, pull_request]
+jobs:
+  fuzzing:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+          Target: [FuzzRangeRequest, FuzzPutRequest, FuzzDeleteRangeRequest]
```

Comment:
Each time when any contributor add a new fuzz case, then they need to update this file?

Reply:
Yes. Unfortunately out-of-box fuzzing requires a specific target method to start the fuzzing. In case you try to use something like `go test -fuzz Fuzz*` the returned error is:

```
testing: will not fuzz, -fuzz matches more than one fuzz test: [FuzzRangeRequest FuzzPutRequest FuzzDeleteRangeRequest]
```
Let me know if there is a better way of doing that

Reply:
I would suggest to get all targets defined in the `scripts/fuzzing.sh ` for now, so that we can manually easily run all fuzz tests locally.

We can update the script/fuzzing.sh something like below,

```
TARGETS="FuzzRangeRequest  FuzzPutRequest  FuzzDeleteRangeRequest"
for target in ${TARGETS}; do
    run pushd ${target_path}
        go test -fuzz ${target}
    run popd
done
```

Reply:
I proceeded by updating the script as requested. Note that, the fuzzing for the different targets won't run in parallel. The fuzz process will start from `FuzzRangeRequest` and finishing with `FuzzDeleteRangeRequest` with the specified timeout